scalatest/src/main/scala/org/scalatest/Engine.scala

```diff
@@ -597,7 +597,7 @@ private[scalatest] sealed abstract class SuperEngine[T](concurrentBundleModMessa
           pos match {
             case Some(pos) => Some(LineInFile(pos.lineNumber, pos.fileName, Some(pos.filePathname)))
             case None =>
-              val stackTraceElements = Thread.currentThread().getStackTrace
+              val stackTraceElements = Thread.currentThread().getStackTrace.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.